### PR TITLE
OLS-186: Cache factory returning Redis interface: unit test

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Cache factory returning Redis interface: unit test

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Unit tests

## Related Tickets & Documents

- Related Issue #[OLS-186](https://issues.redhat.com//browse/OLS-186)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- `make test-unit` should display full code coverage for `CacheFactorry`

### CODE REVIEW

File: tests/unit/test_cache_factory.py [hunk 1]
```diff
@@ -19,6 +21,18 @@ def in_memory_cache_config():
     )
 
 
+@pytest.fixture(scope="module")
+def redis_cache_config():
+    """Fixture containing initialized instance of ConversationCacheConfig."""
+    # os.environ["OLS_CONVERSATION_CACHE"] = constants.REDIS_CACHE
```
Comment (onmete) [inline tests/unit/test_cache_factory.py:27]: Why is the commented line here?

Comment (tisnik) [reply tests/unit/test_cache_factory.py:27]: is shows, how it can be configured on envvar level, see `InMemoryCache` unit tests

Comment (onmete) [reply tests/unit/test_cache_factory.py:27]: I see it there too, but why is this particular information in tests? Especially without further description. It just looks like forgotten code.

Comment (tisnik) [reply tests/unit/test_cache_factory.py]: ok removed @onmete

**Pull request merged**